CHANGELOG.unreleased.md

```diff
@@ -10,6 +10,7 @@
 - [Backup] Merge multiple VHDs at once which will speed up the merging ĥase after reducing the retention of a backup job(PR [#6184](https://github.com/vatesfr/xen-orchestra/pull/6184))
 - [Backup] Implement file cache for listing the backups of a VM (PR [#6220](https://github.com/vatesfr/xen-orchestra/pull/6220))
 - [Backup] Add setting `backups.metadata.defaultSettings.unconditionalSnapshot` in `xo-server`'s configuration file to force a snapshot even when not required by the backup, this is useful to avoid locking the VM halted during the backup (PR [#6221](https://github.com/vatesfr/xen-orchestra/pull/6221))
+- [XO Web] Add ability to configure a default filter for Storage
```

Comment:
```suggestion
- [XO Web] Add ability to configure a default filter for Storage (PR [#6237](https://github.com/vatesfr/xen-orchestra/pull/6237))
```

Reply:
Please also mention #6236 

And also in the PR (`Fixes ....`).